app/helpers/application_helper/toolbar/middleware_domain_center.rb

```diff
@@ -16,4 +16,22 @@ class ApplicationHelper::Toolbar::MiddlewareDomainCenter < ApplicationHelper::To
                    ]
                  ),
                ])
+  button_group('middleware_server_operations', [
+                 select(
+                   :middleware_server_power_choice,
+                   'fa fa-power-off fa-lg',
+                   t = N_('Power'),
+                   t,
+                   :items => [
+                     button(
+                       :middleware_server_stop,
```

Comment:
Glad you asked. I had no idea.
I suppose that we should create a new privilege, `:middleware_domain_stop` ?